civicrm.php

```diff
@@ -1564,6 +1564,14 @@ public function get_request_args() {
       $q = isset($_GET['q']) ? $_GET['q'] : '';
     }
 
+    // Fix 'civicrm/civicrm' elements derived from CRM:url()
```

Comment:
@christianwach Would it makes sense to conditionally load this only if `CIVICRM_CLEANURL` is True?

Reply:
@kcristiano Hmm, good point. Though I can't think of a situation where you'd get a URL of the form:

```
http://domain.tld/civicrm/?page=CiviCRM&q=civicrm%2Fcivicrm%2Fsomething&reset=1&id=1
```

I can't find any instances of this.

Also, I'll have to check if `get_request_args()` is ever called before `CIVICRM_CLEANURL` has been defined.

Reply:
@christianwach I am being overly cautious.   The PR works and should be merged, I was curious if we could tighten it up further.

Reply:
@kcristiano I've done some testing and cannot find an instance where `get_request_args()` is called before `CIVICRM_CLEANURL` has been defined... but I agree that it's always best to be cautious. Hence I've created #153 and #154